todo_cli/todo_CLI.js

```diff
@@ -0,0 +1,106 @@
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    output: process.stdout,
+    input: process.stdin
+})
+
+let listArr = [];
+
+function viewFunc() {
+    let newStr = "";
+    if (listArr.length === 0) {
+        console.log(`List is empty...`);
+    } else {
+        for (let i = 0; i < listArr.length; i++) {
+            newStr = newStr + listArr[i] + "\n";
+        }
+        console.log(newStr);
+    }
+    todoCli();
+}
+
+function newFunc() {
```

Comment:
A function is an action so should be a verb. e.g. `function create() {}`. Also, no need to indicate `Func`